DashAI/back/api/api_v1/endpoints/plugins.py

```diff
@@ -267,3 +267,53 @@ async def update_plugin(
                 status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
                 detail="Internal database error",
             ) from e
+
+
+@router.patch("/{plugin_id}/upgrade")
```

Comment:
We could merge this method with the other patch and make a general one to update plugin parameters. What do you think?

Reply:
I think it is a better practice to keep it separate. In particular considering the open closed principle.